forms_pro/utils/form_generator.py

```
@@ -21,11 +22,17 @@ def create_form_with_doctype(team_id: str, doctype: str):
 def create_form(team_id: str):
     roles = frappe.get_roles(frappe.session.user)
+    print(roles)
     if FORMS_PRO_ROLE not in roles:
         frappe.throw("You are not authorized to create a form")
 
-    form_generator = FormGenerator(team_id=team_id)
-    form_generator.generate()
+    try:
+        form_generator = FormGenerator(team_id=team_id)
+        form_generator.generate()
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)